ci/docker/linux-apt-c-glib.dockerfile

```diff
@@ -53,7 +53,7 @@ RUN luarocks install lgi
 # ERROR: Command errored out with exit status 1: /usr/bin/python3 /usr/share/python-wheels/pep517-0.7.0-py2.py3-none-any.whl/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmpsk4jveay Check the logs for full command output.
 RUN (python3 -m pip install meson || \
          python3 -m pip install --no-use-pep517 meson) && \
-    gem install --no-document bundler
+    gem install --no-document bundler -v 2.4.22
```

Comment:
@kou I am not a huge fan of pinning this here but this fixes CI. Any idea on what we could do? I am not sure if updating Ruby to v3 is worst

Reply:
How about just using Ubuntu 22.04 like GH-39247?

```diff
diff --git a/dev/tasks/tasks.yml b/dev/tasks/tasks.yml
index ed6ea08894..04faef427e 100644
--- a/dev/tasks/tasks.yml
+++ b/dev/tasks/tasks.yml
@@ -1032,6 +1032,8 @@ tasks:
     ci: github
     template: docker-tests/github.linux.yml
     params:
+      env:
+        UBUNTU: 22.04
       image: {{ image }}
 {% endfor %}
 
```

Reply:
Thanks @kou ! Yes, that sounds better. I've pushed the suggestion and triggered the CI jobs.